.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - 'node'
+  - '6'
+  - '4'
+addons:
+  code_climate:
+    repo_token: 'your repo token'
+notifications:
+  slack: feathersjs:T4tvpJFt27wJZABjLSg5OqMv
```

Comment:
Is that token just a remnant of other travis files being copied and pasted? @feathersjs/core-team. In the future we should do a 'slack token' placeholder or just leave that slack portion out since the Travis CLI adds it in, just stick with:

```yaml
notifications:
  email: false